src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -527,7 +526,7 @@ public IntervalList getLociToGenotype(final Collection<Fingerprint> fingerprints
                 final Fingerprint combinedFp = new Fingerprint(specificSample, f, null);
                 fingerprintsByReadGroup.values().forEach(combinedFp::merge);
 
-                final FingerprintResults results = new FingerprintResults(f, specificSample);
+                final FingerprintResults results = new FingerprintResults(f, specificSample, specificSample);
```

Comment:
Yeah, I agree.  That's what the original code was doing, so I left it that way.  I like your idea of changing it to null